DMInRoblox/src/server/ObjectTree/MetaObjects/Part.lua

```diff
@@ -0,0 +1,26 @@
+local objectTree = require(script.Parent.Parent)
+
+return {
+	OnCreate = function(object)
+		object.Part = Instance.new("Part", workspace)
```

Comment:
engineers hate when u do this especially with parts

https://devforum.roblox.com/t/psa-dont-use-instancenew-with-parent-argument/30296

Reply:
its the only way to make your code work but engineers would troll you